docs/src/pages/guides/notifications.md

```diff
@@ -226,9 +226,23 @@ notifications:
 | Type | Notification types                       | `monika-notif`                                                     |
 | Url  | The URL of the Monika Notif Webhook link | `https://whatsapp.hyperjump.tech/api/notify?token=<webhook.token>` |
 
+Please note, to receive messages from the WhatsApp cloud API the following requirements are mandatory.
+
+- The recipient phone number must be a WhatsApp phone number.
+- The recipient must accept [Meta's] new Terms of Service and Privacy Policy.
+- The recipient must use the following WhatsApp version or greater:  
+   Android: 2.21.15.15  
+   SMBA: 2.21.15.15  
+   iOS: 2.21.170.4  
+   SMBI: 2.21.170.4  
+   KaiOS: 2.2130.10  
+   Web: 2.2132.6
+
+See Meta's documentations [here](https://developers.facebook.com/docs/whatsapp/cloud-api/support/error-codes/).
```

Comment:
I dont think we need to show the link to the error codes because our whatsapp service doesn't expose the errors.